badge/badge.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Rohit Prasad" />
+    <meta name="description" content="Badge" />
+    <title>Badge</title>
+    <link rel="stylesheet" href="./badge.css" />
+  </head>
+  <body>
+    <h1>Avatar with badge</h1>
+    <div class="container">
+      <div class="avatar__with__badge">
+        <div class="avatar">
+          <img src="https://img.icons8.com/ios-glyphs/90/000000/user--v1.png" />
```

Comment:
Suggestion: Can add alt attribute for accessibility.

Reply:
Thanks for pointing it out.